unpackaged/config/payment_rollups/classes/CRLP_TEST_VALIDATE_ROLLUPS.cls

```diff
@@ -680,140 +701,7 @@ public class CRLP_TEST_VALIDATE_ROLLUPS {
         rd.npe03__Next_Payment_Date__c = null;
         update rd;
     }
-
```

Comment:
Is the idea that this removed createCustomRollups() method is being replaced by the cci task that sets up the rollups for you?

Reply:
Yes. Most of that cci task metadata already exists. I just added a few extra types to it.